source/material_model/grain_size.cc

```diff
@@ -211,6 +211,51 @@ namespace aspect
 
 
 
+    template <int dim>
+    double
+    GrainSize<dim>::
+    moment_of_grain_size_distribution (const int n) const
+    {
+      // This function normalizes the grain size distribution using the nth moment.
+      // Description can be found in eq 8 of Bercovici and Ricard (2012)
```

Comment:
This type of description should be in the .h file.

Reply:
It would also be good to explain a bit more why we need to do this. Because the use in the function below does not look like it normalizes the grain size (i.e. it gets an unsigned int, not a grain size).